src/components/_basic.less

```diff
@@ -72,3 +72,12 @@ a.sbtn {
 .purple-btn {
   .colorRule(@purple);
 }
+
+.white-btn {
+  .colorRule(@white);
```

Comment:
add color: @darkText

Reply:
is this change needed?

Reply:
Yes. The color of the button is currently white, same color as the button itself. It shouldn't be that way

Reply:
Also, please add a border as the button doesn't really appear in light mode.